docs/sources/flow/reference/components/prometheus.integration.process_exporter.md

```diff
@@ -0,0 +1,92 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.​integration.​process_exporter
+---
+
+# prometheus.integration.process_exporter
+The `prometheus.integration.process_exporter` component embeds
+[process_exporter](https://github.com/ncabatoff/process-exporter) for collecting process stats mined from `/proc`.
+
+## Usage
+
+```river
+prometheus.integration.process_exporter "LABEL"{
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`procfs_path`       | `string`                 | procfs mountpoint. | `/proc` | no
+`track_children`    | `bool`                   | If a proc is tracked, also track any of it's children that are not part of their own group. | `true` | no
+`track_threads`     | `bool`                   | Report on per-threadname metrics as well. | `true` | no
+`gather_smaps`      | `bool`                   | Gather metrics from the smaps file, which contains proportional resident memory size. | `true` | no
+`recheck_on_scrape` | `bool`                   | Recheck process names on each scrape. | `true` | no
+`process_names`     | `process_matcher_config` | A collection of matching rules to use for deciding which processes to monitor. Each config can match multiple processes, which will be tracked as a single process "group."
```

Comment:
I think this 1 goes in a blocks section and gets dropped off this table. Check out another doc with blocks at the top level for an example.

Reply:
+1, we don't put blocks in the arguments section.